metrics/factory.go

```diff
@@ -30,6 +32,12 @@ func NewMetricsRunnerFromDSN(DSN string) (Metrics, contextx.Runner) {
 			WithDDHost(params.Get("host")),
 			WithDDPort(params.Get("port")),
 		)
+	case "datadog-lambda":
+		if namespace == "" {
+			panic("datadog metrics need a namespace")
+		}
+		publisher = NewDataDogLambda()
+		gostats = false
```

Comment:
why always false?

Reply:
the flush interval of the gostats runner is 15 seconds, but a lambda execution should be rather fast so you'd never get to see those metrics.
i thought it was better having a consistent behavior even though we lose the stats metrics, but i'm open to suggestions.